lib/grid_view.rb

```diff
@@ -1,17 +1,28 @@
+require "erb"
```

Comment:
ERB is a standard library, but my goal would be to do this without any libraries. So what would this look like without a rendering library?